src/main/react/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+
+// types
+import type { TodoItemProps } from "../../features/todo/TodoList";
+
+const TabPanel = ({ children, value, index, ...other }) => {
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && <>{children}</>}
+    </div>
+  );
+};
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+const a11yProps = (index: number) => {
```

Comment:
netjes

Reply:
Zeker maar geen credits voor mij: dit is gewoon rechtstreeks uit de Material-UI docs :) 
Ik ben geen fan van Material-UI maar accessiblity doen ze prima!